Zend/tests/final_properties/unset/magic_unset_success.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Test that the __unset() magic method is successful
+--XFAIL--
+Unsetting is currently not possible because of "Cannot modify final property Foo::$properties by reference after initialization"
```

Comment:
Not seeing a reference here, but the fact that it's forbidden is right, no?

Reply:
Ideally, reference shouldn't be involved, so I'll investigate what happens here. Besides, my reasoning is that `$properties` is not initialized, so unsetting it (via `__unset`) should be possible. Or am I missing something?

Reply:
Yeah, you're right, I didn't notice that it's not initialized.